app/routes/auction.ts

```diff
@@ -17,18 +17,36 @@ class AuctionRoute {
     }
 
     index(req: express.Request, res: express.Response) {
-        Auction.find().then(result => {
-            return res.status(200).json({
-                result
+
+        let auctioneer = res.locals.auctioneer;
+        
+        if(auctioneer){
+            
+            Auction.find({ 'auctioneerEmail': res.locals.email }).then(result => {
+                return res.status(200).json({
+                    result
+                });
+            }).catch(error => {
+                console.error(error);
+                return res.status(500).json({ message: 'Failed to retrieve auctions from DB.' })
             });
-        }).catch(error => {
-            console.error(error);
-            return res.status(500).json({ mesasge: 'Failed to retrieve auctions from DB.' })
-        });
+
+        
+        } else {
+            Auction.find().then(result => {
+                return res.status(200).json({
+                    result
+                });
+            }).catch(error => {
+                console.error(error);
+                return res.status(500).json({ message: 'Failed to retrieve auctions from DB.' })
+            });
+        }
     }
```

Comment:
The index route will collect different auction items based on who is connecting.